packages/client/src/components/Login.re

```diff
@@ -94,6 +94,7 @@ let make = (~onLogin) => {
             "Username"->React.string
           </label>
           <input
+            ref={UseAutoFocus.use()}
```

Comment:
Does this feel a bit off? Perhaps it's clearer what it does if we use it closer to the top?

```ocaml
let usernameInputRef = UseAutoFocus.use();

...

<input ref=usernameInputRef .../>
```